cmd/gen/gen.go

```diff
@@ -65,15 +65,50 @@ func main() {
 		0x0303: {name: "document-two", v: map[string]any{"document": 42.13, "foo": "bar"}},
 		0x0304: {name: "document-three", v: map[string]any{"document": int32(0), "baz": nil}},
 
+		// for projection tests
```

Comment:
test_db is a separate project used by others. That comment would be very confusing for them :)

I think the values dataset should be as diverse as possible, but also as small as possible while being useful

Reply:
removed.
also removed documents that aren't in use for tests now.